GitUI/UserControls/BlameControl.cs

```diff
@@ -575,7 +658,10 @@ public GitBlame Blame
             public void BuildAuthorLine(GitBlameLine line, StringBuilder lineBuilder, string dateTimeFormat, string filename, bool showAuthor, bool showAuthorDate, bool showOriginalFilePath, bool displayAuthorFirst)
                 => _control.BuildAuthorLine(line, lineBuilder, dateTimeFormat, filename, showAuthor, showAuthorDate, showOriginalFilePath, displayAuthorFirst);
 
-            public (string gutter, string body) BuildBlameContents(string filename) => _control.BuildBlameContents(filename);
+            public (string gutter, string body, List<GitBlameEntry> avatars) BuildBlameContents(string filename) => _control.BuildBlameContents(filename, 10);
```

Comment:
```csharp
_control.BuildBlameContents(filename, avatarSize: 10);
```
or introduce `const int AvatarSize = 10`, because the meaning of number cannot be immediately seen from usage.